src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let counter = 0;
+  const adder = (a) => {
+    if (a !== undefined) {
```

Comment:
do not compare with undefined

Reply:
why? it's a bad practice?

Reply:
Yes, it is a bad practice, but maybe in this case it is normal, because you can get `0` and it should be a valid value. Let's leave this comparing here 😉 Great solution 🔥